src/components/NewOrder.js

```diff
@@ -55,34 +37,11 @@ class NewOrder extends Component {
   render() {
     return dom.div(
       null,
-      dom.h1(null, 'New Order'),
-      dom.p(
-        null,
-        dom.select(
-          { className: 'select-items form-control input-lg', onChange: this.selectItem },
-          this.props.masterItems.map((item, i) => {
-            return dom.option({ className: 'option', value: item.id, key: i }, item.name);
-          })
-        )
-      ),
-      dom.p(
-        null,
-        dom.input(
-          {
-            className: 'add-comment input-lg form-control',
-            value: this.state.comment,
-            placeholder: 'e.g. No meat, extra sauce',
-            onChange: this.addComment
-          }
-        )
-      ),
-      dom.p(
-        null,
-        dom.button(
-          { className: 'btn btn-default add-entry btn-lg btn-block', onClick: this.addEntry },
-          'Add Item'
-        )
-      ),
+      MasterItems(_.extend({}, {
```

Comment:
Good catch. No reason. Oversight on my part. Same thing for file name. I'll make the change.